doc/commands/group.8.rst

```diff
@@ -133,6 +133,10 @@ Options for ``install``, ``remove`` and ``upgrade``
     | Disable downgrade of dependencies when resolving the requested operation.
     | Used with ``install`` and ``upgrade`` commands.
 
+``--destdir=<path>``
+    | Set directory used for downloading packages to. Default location is to the repository cache directory. Automatically sets the --downloadonly option.
+    | Used with ``install`` and ``upgrade`` commands.
```

Comment:
Since this is not using the shared `destdir.rst` its missing the `Packages downloaded to the destdir are not automatically removed after the next successful transaction.` part.